docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -132,7 +133,7 @@ The Domain `spec` section contains elements for configuring the domain operation
 
 Elements related to domain identification, container image, and domain home:
 
-* `domainUID`: Domain unique identifier. It is recommended that this value be unique to assist in future work to identify related domains in active-passive scenarios across data centers; however, it is only required that this value be unique within the namespace, similarly to the names of Kubernetes resources. This value is distinct and need not match the domain name from the WebLogic domain configuration. Defaults to the value of `metadata.name`.
+* `domainUID`: Domain unique identifier. This identifier is required to be no more than 45 characters. It is recommended that this value be unique to assist in future work to identify related domains in active-passive scenarios across data centers; however, it is only required that this value be unique within the namespace, similarly to the names of Kubernetes resources. This value is distinct and need not match the domain name from the WebLogic domain configuration. Defaults to the value of `metadata.name`. 
```

Comment:
About:  "This identifier is required to be no more than 45 characters." 

A length of 45 requires that WL server and/or cluster names must then be kept unusually short -- so maybe we should advise that the domain-uid be far shorter as a "rule of thumb" both here and in the new 'restrictions' section?   Something like:

"This identifier is strictly required to be no more than 45 characters, and, as a rule of thumb, should be less than ?20? characters in order to help ensure RFC 1123 [restrictions](link to restrictions doc) are met.

Reply:
I think it is a good idea to guide the customers to use a shorter domainUID. But I am not sure about what the magic number is. Ryan, your thoughts? 

Or we could make some general guidance with something like the following(?).

"This identifier is strictly required to be no more than 45 characters, and, practically should be shorter in order to help ensure RFC 1123 [restrictions](link to restrictions doc) are met."

Reply:
Your general guidance suggestion looks good to me.  @rjeberhard Any thoughts?